promise.js

```diff
@@ -1,4 +1,4 @@
-const prom = (item) => {
+const delayedUpperCase = (item) => {
   return new Promise((res, rej) => {
     if (typeof item === 'string') {
```

Comment:
move if statement into setTimeout to not call setTimeout for each if else condiiton

Reply:
I cant see line 8 in your PR something is wrong. Its because you have part of it commited on master already. This is incomplete PR

Reply:
Fix line 8 brackets